### ISSUE TITLE
Unable to compile for Android with Rhomobile 5.5-beta-5

### ISSUE THREAD

**Issue by nhinze** [rhomobile/rhodes#822]
I'm unable to build for Android with Rhomobile-5.5-beta-5. With iOS it works fine. The error message below. This app used to compile just fine with Rhomobile-5.4.

I'm using Android NDK 9d:

build.yml:

```
name: "MyApp"
extensions:
  - json
  - rholang
  - mediacapture
capabilities:
  - gps
#  - camera
  - network_state
  - vibrate
  - sdcard
  - hardware_acceleration
env:
  app: "/Users/me/Documents/AllMyApps/MyApp"
wp:
  productid: "9f59ad90-ad5a-012e-1a31-442c03256517"
sdk: "/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5"
android:
  version: "6.0"
  minSDK: 23
  android_title: 0
  manifest_template: "AndroidManifest.erb"
  abis:
    - arm
    - x86
  emulator: "AVD_6"
build: "release"
applog: "rholog.txt"
version: "3.1.0.0"
vendor: "allmyapps"
sdkversion: "5.5.0"
iphone:
  provisionprofile: 
  BundleURLScheme: "myapp"
  BundleIdentifier: "com.allmyapps.myapp"
  sdk: "iphonesimulator9.3"
# sdk: "iphoneos9.3"
  emulator: 9.3
  emulatortarget: "iphone"
  codesignidentity: 
  entitlements: 
  configuration: "Debug"
```

Error message:

``````
PWD: /Users/me/Documents/AllMyApps/MyApp/bin/tmp/coreapi
CMD: /Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/bin/jar cf /Users/me/Documents/AllMyApps/MyApp/bin/target/android/release/extensions/coreapi/coreapi.jar .
Compiling java sources: ["/var/folders/30/h6wbjjks7bz3ysfq7xchn1b80000gn/T/mediacaptureSRC_build20160825-25346-4renos"]
PWD: /Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5
CMD: /Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/bin/javac -g -d /Users/me/Documents/AllMyApps/MyApp/bin/tmp/mediacapture -source 1.6 -target 1.6 -nowarn -encoding latin1 -classpath /Applications/android-sdk-macosx/platforms/android-23/android.jar:/Users/me/Documents/AllMyApps/MyApp/bin/tmp/.m2/support-annotations-23.0.0.jar:/Users/me/Documents/AllMyApps/MyApp/bin/tmp/.m2/support-v4-23.0.0/libs/internal_impl-23.0.0.jar:/Users/me/Documents/AllMyApps/MyApp/bin/tmp/.m2/support-v4-23.0.0/classes.jar:/Users/me/Documents/AllMyApps/MyApp/bin/tmp/Rhodes:/Users/me/Documents/AllMyApps/MyApp/bin/target/android/release/extensions/coreapi/android-support-v4.jar:/Users/me/Documents/AllMyApps/MyApp/bin/target/android/release/librhodes/Rhodes.jar:/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/platform/android/lib/commons-codec-1.9.jar:/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/platform/android/lib/commons-logging-1.2.jar:/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/platform/android/lib/fluent-hc-4.5.jar:/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/platform/android/lib/httpclient-4.5.jar:/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/platform/android/lib/httpclient-cache-4.5.jar:/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/platform/android/lib/httpclient-win-4.5.jar:/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/platform/android/lib/httpcore-4.4.1.jar:/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/platform/android/lib/httpmime-4.5.jar:/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/platform/android/lib/jna-4.1.0.jar:/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/platform/android/lib/jna-platform-4.1.0.jar:/Users/me/Documents/AllMyApps/MyApp/bin/target/android/release/extensions/coreapi/coreapi.jar @/var/folders/30/h6wbjjks7bz3ysfq7xchn1b80000gn/T/mediacaptureSRC_build20160825-25346-4renos
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

PWD: /Users/me/Documents/AllMyApps/MyApp/bin/tmp/mediacapture
CMD: /Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/bin/jar cf /Users/me/Documents/AllMyApps/MyApp/bin/target/android/release/extensions/mediacapture/mediacapture.jar .
-$TIME$- message [ build:android:extensions_java FINISH ] time is { 2016-08-25T23:02:04Z } milliseconds from start (494305)
-$TIME$- message [ package:android START ] time is { 2016-08-25T23:02:04Z } milliseconds from start (494306)
Running dx utility
PWD: /Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5
CMD: /Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/bin/java -Xmx1024m -jar /Applications/android-sdk-macosx/build-tools/23.0.2/lib/dx.jar --dex --output=/Users/me/Documents/AllMyApps/MyApp/bin/classes.dex /Users/me/Documents/AllMyApps/MyApp/bin/target/android/release/extensions/coreapi/android-support-v4.jar /Users/me/Documents/AllMyApps/MyApp/bin/target/android/release/extensions/coreapi/coreapi.jar /Users/me/Documents/AllMyApps/MyApp/bin/target/android/release/extensions/mediacapture/mediacapture.jar /Users/me/Documents/AllMyApps/MyApp/bin/target/android/release/librhodes/Rhodes.jar /Users/me/Documents/AllMyApps/MyApp/bin/tmp/.m2/support-annotations-23.0.0.jar /Users/me/Documents/AllMyApps/MyApp/bin/tmp/.m2/support-v4-23.0.0/libs/internal_impl-23.0.0.jar /Users/me/Documents/AllMyApps/MyApp/bin/tmp/.m2/support-v4-23.0.0/classes.jar
Uncaught translation error: java.lang.IllegalArgumentException: already added: Landroid/support/annotation/AnimRes;
Uncaught translation error: java.lang.IllegalArgumentException: already added: Landroid/support/annotation/AnimatorRes;
Uncaught translation error: java.lang.IllegalArgumentException: already added: Landroid/support/annotation/AnyRes;
Uncaught translation error: java.lang.IllegalArgumentException: already added: Landroid/support/annotation/ArrayRes;
Uncaught translation error: java.lang.IllegalArgumentException: already added: Landroid/support/annotation/AttrRes;
Uncaught translation error: java.lang.IllegalArgumentException: already added: Landroid/support/annotation/BinderThread;
Uncaught translation error: java.lang.IllegalArgumentException: already added: Landroid/support/annotation/BoolRes;
Uncaught translation error: java.lang.IllegalArgumentException: already added: Landroid/support/annotation/CallSuper;
Uncaught translation error: java.lang.IllegalArgumentException: already added: Landroid/support/annotation/CheckResult;

UNEXPECTED TOP-LEVEL EXCEPTION:
java.lang.RuntimeException: Translation has been interrupted
 at com.android.dx.command.dexer.Main.processAllFiles(Main.java:608)
 at com.android.dx.command.dexer.Main.runMonoDex(Main.java:311)
 at com.android.dx.command.dexer.Main.run(Main.java:277)
 at com.android.dx.command.dexer.Main.main(Main.java:245)
 at com.android.dx.command.Main.main(Main.java:106)
Caused by: java.lang.InterruptedException: Too many errors
 at com.android.dx.command.dexer.Main.processAllFiles(Main.java:600)
 ... 4 more
rake aborted!
Error running DX utility
/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/platform/android/build/android.rake:2219:in `block (2 levels) in <top (required)>'
/Users/me/.rvm/gems/ruby-1.9.3-p551/gems/rhodes-5.5.0.beta.5/Rakefile:4291:in `invoke'
/Users/me/.rvm/gems/ruby-1.9.3-p551/bin/ruby_executable_hooks:15:in `eval'
/Users/me/.rvm/gems/ruby-1.9.3-p551/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => run:android => run:android:emulator => run:android:emulator:build => device:android:debug => package:android
(See full trace by running task with --trace
```)
``````

**Comment by rognar** [comment]
Hi,

Please Sign-Up(using your developer e-mail) at tautechnologies.zendesk.com. After that you will be able to create tickets.

Regards,
Konstantin

**Comment by rognar** [comment]
The issue is fixed in Rho 5.5 beta 7. You can find it here -http://rhomobile.tau-technologies.com/list.html
